cetmix_tower_server/models/cx_tower_command.py

```diff
@@ -140,6 +140,27 @@ class CxTowerCommand(models.Model):
         column2="variable_id",
     )
 
+    if_file_exists = fields.Selection(
+        selection=[
+            ("skip", "Skip"),
+            ("overwrite", "Overwrite"),
+            ("raise", "Raise Error"),
+        ],
+        string="If File Exists",
```

Comment:
No need to add a string here explicitly, it will be [generated automatically](https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#fields) from the field name.